src/bodywork/k8s/deployments.py

```diff
@@ -358,18 +358,22 @@ def monitor_deployments_to_completion(
 
 def list_service_stage_deployments(
     namespace: Optional[str] = None,
+    name: Optional[str] = None,
```

Comment:
This is what I was saying the other day, deployment name and project name are the same thing. It would be nice to stick to one name instead of two but we can discuss this in person at a later date. 

We have at least managed to make things clearer than before in this iteration.

Reply:
Agreed.